test-gui.sh

```diff
@@ -8,7 +8,7 @@ yarn install
 if [ -z "${CHROME_VERSION-}" ]; then
  yarn run webdriver-update
 else
- yarn run webdriver-update --versions.chrome="$CHROME_VERSION"
+ yarn run webdriver-update --versions.chrome="$CHROME_VERSION" --gecko=false
```

Comment:
You probably should be checking `BRIDGE_E2E_BROWSER_NAME` instead. If not `firefox`, skip gecko.

https://github.com/spadgett/console/blob/01cc9d3980936a7227ede924bc024c97c38fd033/frontend/integration-tests/protractor.conf.ts#L18

Reply:
Updated, is it better this way?